src/dirtyfields/dirtyfields.py

```diff
@@ -42,7 +42,7 @@ def _as_dict(self, check_relationship, include_primary_key=True):
         deferred_fields = self.get_deferred_fields()
 
         for field in self._meta.fields:
-            if self.FIELDS_TO_CHECK and (field.get_attname() not in self.FIELDS_TO_CHECK):
```

Comment:
@MounirMesselmeni as explained in the discussion on this PR, can we make it backward compatible by allowing both with and without `_id`?
Thanks!